src/main/scala/org/apache/spark/sql/hive/thriftserver/TiSparkSQLDriver.scala

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
This is almost a copy of original spark with minor changes.
I suppose we keep original copyright?